### PR TITLE
Feat dev stage

### LINKED ISSUE #32: Setup one AWS/serverless environment per dev

### PR DESCRIPTION
**Pull request by MollardMichael**
You can look through the README to see what I did in details but to put it simply, I'm using the stage variable to push a separate environment for each developer.

What do you think about it ?

Closes #32

### CODE REVIEW

File: backend/libs/response.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        body: JSON.stringify(body),
+    };
+}
+
+export const success = async (body?: Body) => {
+    return buildResponse(200, body);
+};
+
+export const failure = async (body?: Body) => {
+    return buildResponse(500, body);
```
Comment (mourtisma) [inline backend/libs/response.ts:43]: What happens if the API Gateway fails with a code other than 500 ?

Comment (MollardMichael) [reply backend/libs/response.ts:43]: Nothing. As we're using websocket, the status code is not event that useful.

If you send a status code it is usually use as the response status code to the incoming request

File: README.md [hunk 2]
```diff
@@ -45,6 +45,26 @@ Run `make install` in root directory to install serverless
...
+ex: for me
+
+```bash
+$ id -un
+michaelm
+```
+
+So If I run make deploy, a stage named michaelm will be used.
+
+This imply that
```
Comment (Startouf) [inline README.md:62]:
```suggestion
This implies
```

File: README.md [hunk 3]
```diff
@@ -45,6 +45,26 @@ Run `make install` in root directory to install serverless
...
+
+-   A new API Gateway stage named michaelm will be created
+-   All the function will be deployed with the according to the naming convention watch-with-me-michaelm-funciton_name
```
Comment (Startouf) [inline README.md:65]:
```suggestion
-   Functions will be deployed according to the naming convention watch-with-me-michaelm-function_name
```

File: backend/Makefile [hunk 4]
```diff
@@ -7,28 +9,34 @@ install: install-deps ## Install all dependencies
...
+configure: ## Prompt you to configure you AWS Environement
 	aws configure --profile lesgars-watch-with-me
 
-format:
+format: ## Run eslint and prettier for formatting
 	pnpm run lint:fix
 	pnpm run prettier:fix
 
-build:
+build: ## Run webpack build process (Usefull for testing)
```
Comment (Startouf) [inline backend/Makefile:26]:
```suggestion
build: ## Run webpack build process (Useful for testing)
```

**Pull request merged**